test/WebJobs.Script.Tests/Workers/Rpc/GrpcWorkerChannelTests.cs

```diff
@@ -939,6 +939,45 @@ public async Task ReceivesInboundEvent_FunctionLoadResponse()
             Assert.True(traces.Any(m => string.Equals(m.FormattedMessage, "Received FunctionLoadResponse for function: 'js1' with functionId: 'TestFunctionId1'.")), "FunctionLoadResponse TestFunctionId1");
         }
 
+        [Fact]
+        public async Task Receives_Individual_FunctionLoadResponses_Parallel()
+        {
```

Comment:
Did you confirm this was failing before the change?

Reply:
I even tried 1000 functions, and multiple local test run could not repro this issue because it is such a rare race condition to repro).